dlk/python/dlk/core/view.py

```diff
@@ -744,6 +744,8 @@ def run(self):
 
             return self.format_string(f"""func_Lookup({inputs_string}, {op.name});""")
 
+        raise TypeError(f"{self.op.op_type} is not supported in view.py!")
```

Comment:
This error message is not telling a lie, but "not supported in View.run" whould be more kind.

Also, the exclamation mark emphasis should not be used casually.

Reply:
I have fixed it.